icons/monzo.svg

```diff
@@ -0,0 +1,2 @@
+<svg role ="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><title>Monzo icon</title><path d="M12 17l6-6v11l1 1 5-5a1 1-180 0 0 0-1V5l-4-4h-1l-7 8-7-8H4L0 5v12a1 1-180 0 0 0 1l5 5 1-1V11l6 6z"/></svg>

```

Comment:
The optimization you ended up doing was a bit too harsh as you can see in the side-by-side comparision ⬇️ I suggest using this version, obtained with a precision of 2 in [SVGOMG](https://jakearchibald.github.io/svgomg/):

```suggestion
<svg role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><title>Monzo icon</title><path d="M11.99 17.26H12l5.93-5.92v11.04a.45.45-180 0 0 .76.31l4.98-4.97a1.11 1.11-180 0 0 .32-.8V5.29l.01-.01-3.98-3.97a.44.44-180 0 0-.63 0L12 8.69l-7.41-7.4a.44.44-180 0 0-.63 0L0 5.28h.01H0v11.66a1.11 1.11-180 0 0 .32.79l4.97 4.98a.45.45-180 0 0 .76-.32l.01-11.06 5.93 5.94z"/></svg>
```

![image](https://user-images.githubusercontent.com/3742559/50853429-7fa97900-1382-11e9-8ac8-7a3128df8f19.png)

Reply:
@d3xt3r1232 it seems you have made  the change and then reverted it (see ⬇️) Can you try updating the SVG once again? If you need help with git or anything related, feel free to ask me 🙂 

![image](https://user-images.githubusercontent.com/3742559/50885511-31ca5a80-13ee-11e9-8c01-92a2a5b0359c.png)

Reply:
@ericcornelissen maybe it is a bad question, but how to commit the same file again?

Reply:
@ericcornelissen i had some problems with my git bash, is the icon fine now?

Reply:
@d3xt3r1232, the icon still [looks like ⬆️](https://github.com/simple-icons/simple-icons/pull/1155#discussion_r246288591).

If you want to update the SVG I recommend you copy the one I provided in [my comment above](https://github.com/simple-icons/simple-icons/pull/1155#discussion_r246119738) and just commit it again, e.g.:

```bash
# Add the icon
git add icons/monzo.svg

# Commit the change
git commit -m "Update Monzo icon SVG"

# Push the new commit (in your case to the "develop" branch)
git push origin develop
```

Let me know if that works or not, including the output you get in git bash.

Reply:
Sorry, @ericcornelissen, my fault :/. I uploaded the monzo icon again.